### ISSUE TITLE
Linux Sandcastle on Odysseyn1x unable to connect to device

### ISSUE THREAD

**Issue by ghost** [raspberryenvoie/odysseyn1x#4]
When I use Linux Sandcastle, it boots to PongoOS normally and then as it boots Linux, it says “Unable to apply overlay FAT” on the PongoOS command line screen on my iPad 6. I don’t know what this means, and I also don’t know why Odysseyn1x says “Success!” because it boots back to stock iOS just like if I rebooted, strangely. It may just be my iPad.

**Comment by raspberryenvoie** [comment]
I don't think Project Sandcastle supports the iPad.
https://projectsandcastle.org/status

**Comment by ghost** [comment]
I honestly don’t know, but when I try an iPod Touch, it just says lost device connection or something like that. I got the idea about Project Sandcastle on iPad from [here](https://iPadLinux.org)

**Comment by raspberryenvoie** [comment]
Does Project Sandcastle support your iPod Touch?
Concerning your iPad, I don't know, never tried Project Sandcastle on an iPad. I let you close this issue if you think it's okay.

**Comment by ghost** [comment]
Yeah my iPod Touch is a iPod 6 and it supports Project Sandcastle. I want to try Project Sandcastle on an iPhone 7, but I just have to get one first! I just changed the issue to my iPod touch for now.

**Comment by ghost** [comment]
When I retry Android Sandcastle on my iPod I’ll take a picture and put it on here so you can see what it says.

**Comment by ghost** [comment]
Sorry I haven’t checkra1ned I’m a while, but now that checkra1n supports iPadOS 14, I’ll have to try it! :)
Edit: maybe in a few weeks because it doesn’t support my iPad chip at the moment.

**Comment by raspberryenvoie** [comment]
Okay :)

**Comment by Anwu949** [comment]
Any way to update Project Sandcastle to work on A8 devices?

**Comment by ghost** [comment]
> Any way to update Project Sandcastle to work on A8 devices?

That I don’t know. There a guide photo on the Project Sandcastle website, so you might want to check that.

**Comment by 112cxyz** [comment]
i get the error on a iphone 7

**Comment by raspberryenvoie** [comment]
@Tsumiicode Can you provide more info pls?

**Comment by raspberryenvoie** [comment]
> Yeah my iPod Touch is a iPod 6 and it supports Project Sandcastle. I want to try Project Sandcastle on an iPhone 7, but I just have to get one first! I just changed the issue to my iPod touch for now.

Your iPod 6 isn't supported. From https://projectsandcastle.org/status:
> Currently, these builds are only supported for iPhone 7 / 7+ and the iPod touch 7. Please see below for a full list of supported feature

Anyway @ghost is a deleted account so the original author of this issue will propably never see this.

**Comment by reoccurcat** [comment]
> Anyway @ghost is a deleted account so the original author of this issue will propably never see this.

Don't worry, I just created a new account. Yeah, I wish the iPod 6 was supported, but I should get my hands on an iPhone 7 pretty soon.

**Comment by raspberryenvoie** [comment]
Glad you're back ^^

**Comment by reoccurcat** [comment]
:)

**Comment by 112cxyz** [comment]
Right so sorry I didn’t reply my emails where a bit bugged and GitHub
wouldn’t let me login but it just fails to boot it just normally just sits
there or just shows a normal pongo shell

**Comment by raspberryenvoie** [comment]
> Right so sorry I didn’t reply my emails where a bit bugged and GitHub
> wouldn’t let me login but it just fails to boot it just normally just sits
> there or just shows a normal pongo shell

Thank you. Is there an error message displayed on the computer?

**Comment by 112cxyz** [comment]
Normaly it just says no device found or a usb error

**Comment by raspberryenvoie** [comment]
Sorry idk how to solve that. Does anyone know?

**Comment by 112cxyz** [comment]
I’ve tried on 3 devices but recently checkra1n has refused to jb my iPhone
7 but my 6 is fine? So idk but I don’t use project sandcastle on it because
that would be dumb

**Comment by raspberryenvoie** [comment]
If checkra1n can't jb your iPhone 7, that might be the reason

**Comment by 112cxyz** [comment]
I’ve been looking into it and there has been a occurring issue with
checkra1n not claiming the device on boot so it errors with a code -33 or
-31

**Comment by raspberryenvoie** [comment]
Maybe open an issue on the checkra1n bugtracker
